### ISSUE TITLE
Mobile site homepage banner

### ISSUE THREAD

**Issue by thelinkscholars** [thelinkscholars/thelinkscholars.github.io#11]
There's an issue with the mobile site banner. The aspect ratio is off and so the picture is cut off and most of the picture isn't visible

**Comment by HexRdesign** [comment]
Which banner is this, the first or second? I'm guessing it's the second?

**Comment by thelinkscholars** [comment]
yes it's the second banner with everyone on the team and the mentors

**Comment by HexRdesign** [comment]
I think the same workaround used in the first banner should be used (create a mobile-specific banner) - given how wide the banner is, there isn't really a feasible way to get everyone to fit on smaller screens
